setup.py

```diff
@@ -12,7 +12,7 @@
     history = history_file.read()
 
 install_requires = [
-    'Faker>=3.0.0,<10',
+    'Faker>=3.0.0,<15',
```

Comment:
Can you change this in the conda file as well?

Reply:
I'm actually not sure if the conda file will work for this because I'm not sure if we ever released rdt >= 1.0.0 on our conda channel. I think we've just been releasing on conda forge